app/views/abuse_reports/new.html.erb

```diff
@@ -8,71 +8,121 @@
 
 <!--main content-->
 <div class="userstuff">
-  <p><%= t(".purview.about",
-            tos_link: (link_to t(".purview.tos"), tos_path),
-            support_link: (link_to t(".purview.support"), new_feedback_report_path)).html_safe %></p>
-  <p><%= t(".do_not_spam_html") %></p>
+  <p class="important" name="reportwhat">
+    <%= t(".purview.about_html",
+          tos_link: (link_to t(".purview.tos"), tos_path)) %>
+  </p>
+
+  <p>
+    <%= t(".purview.technical_help_html",
+          support_link: (link_to t(".purview.support"), new_feedback_report_path)) %>
+  </p>
+
   <p>
-    <%= t(".reportable.intro") %>
+    <%= t(".purview.dmca_takedown_html",
+          dmca_link: (link_to t(".purview.dmca"), dmca_path)) %>
   </p>
+
+  <p name="reportwhy">
+    <strong>
+      <%= t(".reportable.intro") %>
+    </strong>
+  </p>
+
   <ul>
+    <li><%= t(".reportable.violation_html",
+              tos_link: (link_to t(".purview.tos"), tos_path, name: "reportwhat")) %></li>
+    <li><%= t(".reportable.content") %></li>
+    <li><%= t(".reportable.harassment") %></li>
     <li><%= t(".reportable.hack") %></li>
-    <li><%= t(".reportable.tos") %></li>
-    <li><%= t(".reportable.dmca.claim", dmca_link:
-            (link_to t(".reportable.dmca.policy"), dmca_path)).html_safe %></li>
+    <li><%= t(".reportable.suspended") %></li>
+  </ul>
+
+  <p name="reporthow">
+    <strong>
+      <%= t(".include.intro") %>
+    </strong>
+  </p>
+
+  <ul>
+    <li><%= t(".include.username") %></li>
+    <li><%= t(".include.links") %></li>
+    <li><%= t(".include.quote_html",
+              tos_link: (link_to t(".include.ao3"), tos_path(anchor: 'content'))) %></li>
```

Comment:
Can you use double quotes on the `anchor`, or is there a reason single quotes are needed?

Reply:
Apologies, I come from JS. I'll keep a better eye out for it.